core/api/transport/impl/session_impl.cpp

```diff
@@ -21,42 +22,41 @@ namespace kagome::server {
   void SessionImpl::stop() {
     stopTimer();
     socket_.cancel();
-    onStopped()(shared_from_this());
   }
 
   void SessionImpl::asyncRead() {
     boost::asio::async_read_until(
-        socket_, buffer_, '\n',
+        socket_,
+        buffer_,
+        '\n',
```

Comment:
do we still read until \n?

Reply:
SessionImpl class is not required and can be removed, I left it just in case, since it works

Reply:
remove it. it will be in git history, so we can restore it if needed

Reply:
removed